qemu/tests/pxe_query_cpus.py

```diff
@@ -85,7 +85,12 @@ def run(test, params, env):
             count += 1
             try:
                 vm.monitor.info("cpus", debug=False)
-                vm.verify_status("running")
+                if params.get('machine_type').startswith("s390"):
+                    if 'running' or 'guest-panicked' in \
+                            vm.monitor.get_status()['status']:
+                        pass
```

Comment:
It will always pass.
```python
'running' or 'guest-panicked' in 'foo'
'running'
```
Another one, when the status is not 'running' or 'guest-panicked', needs to raise it.

Reply:
Modified

Reply:
Seems the problem has not solved yet.
`running' or 'guest-panicked' in 'foo` will always return `running` (means always `True`)

Reply:
modified